CRM/Mailing/BAO/Mailing.php

```diff
@@ -282,8 +282,10 @@ public static function getRecipients($mailingID) {
     // Get recipients selected in prior mailings
     if (!empty($priorMailingIDs['Include'])) {
       CRM_Utils_SQL_Select::from('civicrm_mailing_recipients')
-        ->select("contact_id, $entityColumn")
+        ->select("civicrm_mailing_recipients.contact_id, $entityColumn")
```

Comment:
can you also add DISTINCT to select clause mostly on queries which involve temp table? because lately I encountered a issue that temp table are populated with duplicate entries and this causes performance issue in menu rebuild

Reply:
@monishdeb sure

Reply:
As you can [here](url
https://github.com/civicrm/civicrm-core/blob/4.6/CRM/Mailing/BAO/Mailing.php#L211) the old code got DISTINCT in select clauses and this change has been lost during refactoring changes :(

Reply:
@monishdeb added now